reference-data/kartothek-compat/py2-dataset/src/create_dataset.py

```diff
@@ -0,0 +1,29 @@
+import sys
+from distutils.version import LooseVersion
+from functools import partial
+
+import pandas as pd
+import storefact
+
+import kartothek
+from kartothek.io.eager.write import store_dataframes_as_dataset  # noqa
+
+assert sys.version_info.major == 2  # Show we're on Python 2
+assert LooseVersion(kartothek.__version__) < "3"  # Tested using `2.31.4`
```

Comment:
There is no public `2.31.4`

Reply:
I think we should not check in this file anyhow. Try creating the test data during runtime by defining the cols as `b"col_name"`

Reply:
Do you consider it more appropriate to simulate the previously-incompatible data, instead of using a dataset with identical characteristics as the one in the original bug report?

Reply:
We cannot base anything on non-public releases.

Reply:
Fair enough